__tests__/app.test.js

```diff
@@ -76,7 +76,61 @@ describe('/api/articles/:article_id', () => {
             expect(response.body.status).toBe(400)
             expect(response.body.msg).toBe('Invalid ID parameter')
         })
-    })
+    });
+    test('PATCH 201: updates the value of votes on an article with a matching article_id', () => {
+        const patchObj = { inc_votes : 10 }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(patchObj)
+        .expect(200)
+        .then((response) => {
+            const { article } = response.body
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String)
+            })
+            expect(article.votes).toBe(110)
+        })
+    });
+    test('PATCH 400: Respond with an appropriate status code and error message when passed invalid patch request data', () => {
+        const patchObj = { bad_data : 'I\'m bad data' }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(patchObj)
+        .expect(400)
+        .then((response) => {
+            expect(response.body.status).toBe(400)
+            expect(response.body.msg).toBe('Bad request')
```

Comment:
Ideally this would be Status 200, if inc_votes is missing from patch object (and you include other meaningless properties). It should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)